14_contains/README.md

```diff
@@ -0,0 +1,3 @@
+# Exercise 14 - contains
+
+Write a function that searches for a value in a nested object. It returns true if the object contains that value.
```

Comment:
1. I think we should make it explicit that objects are compared by reference.
2. Like other exercises, some basic examples will help.

Reply:
Ok, I added some basic examples. If someone wants to view all cases they can always look at the tests